RecipeRoulette/RecipeRoulette
# Updates to Frontend : React-Router-Dom, Search bar w/ Search Options

*** What is the feature/fix?
Previously did not have a way to send search options along with the search bar query. Previously were not able to change pages upon successful login of user. 

*** What is the solution?
Utilized Styled-Components, MUI library, and React-Hook-Form to create styled div for accepting search options to be sent along with the search bar query. Utilized React-Router-Dom to set up a Browser Router and created Routes for different pages and a Navbar to be displayed on all pages. 

*** What areas of the site does it impact?
The entirety of the frontend. 

*** How to test?
At the login page attempt to login and if login is successful should be navigated to the home page. On any page attempt a search with or without search options and see that the server receives search query and search options. 

*** Other Notes
Requires running of npm install for MUI package.
